apps/mobile/app/screens/Authenticated/TeamScreen/components/ListCardItem.tsx

```diff
@@ -148,13 +151,23 @@ const ListCardItem: React.FC<Props> = observer((props) => {
 				...$listCard,
 				...GS.mt5,
 				paddingTop: 4,
-				backgroundColor: !props.member?.employee?.isActive
-					? "suspended"
-					: props.member?.employee?.isOnline || props.member?.timerStatus === "running"
-					? "#9FDAB7"
-					: !props.member?.totalTodayTasks?.length
-					? "#F1A2A2"
-					: "#EBC386",
+				backgroundColor:
+					!timerStatus?.running &&
+					timerStatus?.lastLog &&
+					timerStatus?.lastLog?.startedAt &&
+					moment().diff(moment(timerStatus?.lastLog?.startedAt), "hours") < 24 &&
+					(timerStatus?.lastLog?.source !== "MOBILE" || props.member?.employee?.isOnline)
```

Comment:
@desperado1802 I don't get this line:

If member is isOnline at some other source then we will saw `Pause`?

Reply:
Also @desperado1802 If you see in video, not only current member's but all other member's status is shown "Paused".